dist/tools/cppcheck/check.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/sh
 
+# Copyright 2017 Kaspar Schleiser <kaspar@schleiser.de>
 # Copyright 2014 Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
 # Copyright 2014 Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
 #
 # This file is subject to the terms and conditions of the GNU Lesser
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
+. ${RIOTBASE:+${RIOTBASE}/}dist/tools/ci/changed_files.sh
```

Comment:
uff. We need to come up with "style commenting guidelines". Those subjective "I prefer style X over Y" IMO need to get some substance.

Your argument could also be used against usage of C's ternary operator.